integration_test/sql/migrator.exs

```diff
@@ -36,18 +59,14 @@ defmodule Ecto.Integration.MigratorTest do
 
   import Ecto.Migrator
```

Comment:
Minor nitpick: should this be moved to the top where the line `import Ecto.Migrator, only: [migrated_versions: 1]` was?